### PR TITLE
Mailjet integration

### PR DESCRIPTION
**Pull request by irohit427**

### CODE REVIEW

File: mail/mail.go [hunk 1]
```diff
@@ -5,12 +5,17 @@ import (
 	"time"
 
 	"github.com/mailgun/mailgun-go"
+	mailjet "github.com/mailjet/mailjet-apiv3-go"
```
Comment (Hemant-Mann) [inline mail/mail.go:8]: add this in the readme file @irohit427

File: mail/mail.go [hunk 2]
```diff
@@ -40,3 +52,31 @@ func SendViaMailgun(conf *Config, params *Params) (string, string, error) {
...
+		  },
+		  Subject: params.Subject,
+		  TextPart: params.TextPart,
+		  HTMLPart: params.HTMLPart
+		},
+	}
+	messages := mailjet.MessagesV31{Info: messagesInfo}
+	res, err := mailjetClient.SendMailV31(&messages)
+	if err != nil {
+		log.Fatal(err)
```
Comment (Hemant-Mann) [inline mail/mail.go:79]: you can not log error inside a function, you need to return it

File: mail/mail.go [hunk 3]
```diff
@@ -19,6 +24,13 @@ type Params struct {
 	Timeout         int      // timeout in seconds
 }
 
+type MailjetParams struct {
+	SenderEmail, SenderName string
+	RecipientEmail, RecipientName string
+	Subject string
+	TextPart, HtmlPart string
```
Comment (Hemant-Mann) [inline mail/mail.go:31]: there should be option to add CC (Ref: line 23) and Reply To

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]
ReplyTo is still not added

**Comment by irohit427** [comment]
> ReplyTo is still not added

Added ReplyTo

File: mail/mail.go [hunk 4]
```diff
@@ -40,3 +54,48 @@ func SendViaMailgun(conf *Config, params *Params) (string, string, error) {
...
+
+	messagesInfo := []mailjet.InfoMessagesV31 {
+		mailjet.InfoMessagesV31{
+		  From: &mailjet.RecipientV31{
+			Email: params.SenderEmail,
+			Name: params.SenderName,
+		  },
+		  ReplyTo: &mailjet.RecipientV31{
+			  Email: params.ReplyToEmail,
+		  }
```
Comment (Hemant-Mann) [inline mail/mail.go:89]:
```suggestion
		  },
```

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: mail/mail.go (hunk 4 above)
Comment (irohit427) [reply mail/mail.go:89]: resolved

File: mail/mail.go [hunk 5]
```diff
@@ -40,3 +54,48 @@ func SendViaMailgun(conf *Config, params *Params) (string, string, error) {
...
+		  },
+		  ReplyTo: &mailjet.RecipientV31{
+			  Email: params.ReplyToEmail,
+		  },
+		  To: toMailjetRecepient,
+		  CC: ccMailjetRecepient,
+		  BCC: bccMailjetRecepient,
+		  Subject: params.Subject,
+		  TextPart: params.TextPart,
+		  HTMLPart: params.HTMLPart
```
Comment (Hemant-Mann) [inline mail/mail.go:95]:
```suggestion
		  HTMLPart: params.HTMLPart,
```

**Review by Hemant-Mann — APPROVED** [review]

**Pull request merged**